CHANGELOG.rst

```diff
@@ -3,19 +3,22 @@
 0.52.0
 ------
 
-Renku ``0.52.0`` contains a new secrets-storage service that allows users to store and use secrets in 
-sessions.
-
+Renku ``0.52.0`` introduces new features related to Renku 2.0. However, Renku 2.0 is still
+in early development and is not yet accessible to users. For more information, see our
+[roadmap](https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/roadmap.md).
 
 **🌟 New Features**
 
 - **Data Services**: Added new secrets storage service for managing user session secrets, including
   new endpoints on data-service to manage these secrets.
+- **Data Services**: Updated the authorization for access to Renku 2.0 projects to work with Authzed DB,
+  a 3rd party database dedicated to saving authorization data and making authorization decisions
```

Comment:
We're missing the User-Facing Changes and Internal Changes sections. Please put the secrets line in the User-Facing Changes section and the Authorization line under Internal Changes (since users won't be able to tell the difference).